Source/Engines/ModuleEnginesRF.cs

```diff
@@ -54,7 +54,7 @@ public class ModuleEnginesRF : ModuleEnginesSolver
 
         protected Propellant curveProp;
 
-        [KSPField(guiName = "#RF_EngineRF_IgnitedFor", guiUnits = "s", guiFormat = "F3", groupName = groupName)] // Ignited for 
+        [KSPField(guiName = "#RF_EngineRF_IgnitedFor", guiUnits = "s", guiFormat = "F3", groupName = groupName)] // Ignited for
```

Comment:
Whole bunch of weird whitespace changes in the diff. Did you run it through an autoformatter or something?

Reply:
Edited it in vim and I've got a whole bunch of autoformat-on-save rules to kill whitespace issues

Reply:
Yeah it's sane enough to do that and on double-checking the rest I'm not worried about the amount of diff so...resolving this. 👍